src/main/java/data/CameraTimeline.java

```diff
@@ -1,31 +1,95 @@
 package data;
 
-import java.util.ArrayList;
-
 import lombok.Getter;
 import lombok.Setter;
 
+import java.util.ArrayList;
+import java.util.LinkedList;
+
 /**
- * Created by Bart.
  * Class to store information about a camera timeline.
+ * @author Bart.
  */
 public class CameraTimeline extends Timeline {
 
     // The camera that is associated with this timeline.
-    @Getter @Setter
+    @Getter
```

Comment:
Until now we have used @Getter and @Setter on the same line. Let's keep it the same everywhere